l2switch-main/impl/src/main/java/org/opendaylight/l2switch/flow/chains/ServiceChain.java

```diff
@@ -155,8 +185,8 @@ private void restartContProcess(String contName) {
 	return ncrs;
     }
     
-    public NodeConnectorRef[] startAccessibleCont_getNCR(String contName, String contImage, String[] ifaces, String ip) {
-	this.containerCalls.startContainer(contName, contImage, this.devNum);
+    public NodeConnectorRef[] startAccessibleCont_getNCR(String contName, String contImage, String[] ifaces, String ip, String iot_IP) { //nmap
+	this.containerCalls.startContainer(contName, contImage, this.devNum, iot_IP);
```

Comment:
so do you have to provide the target IP address now?

Reply:
for both X and A type containers, yes